x-pack/legacy/plugins/canvas/public/components/keyboard_shortcuts_doc/keyboard_shortcuts_doc.tsx

```diff
@@ -46,7 +49,7 @@ const getDescriptionListItems = (shortcuts: ShortcutMap[]): DescriptionListItem[
           // replace +'s with spaces so we can display the plus symbol for the plus key
           shortcut = shortcut.replace(/\+/g, ' ');
           if (i !== 0) {
-            acc.push(<span key={getId('span')}> or </span>);
+            acc.push(<span key={getId('span')}> {strings.getShortcutSeparator()} </span>);
```

Comment:
I'm pretty sure this need to be refactored in some way to support:

```
defaultValue: "{shortcut1}, {shortcut2} or {shortcut3}"
values: {...}
```

Reply:
Clint's suggestion here is the right one, rather than having "or" as a separate string on its own.  His example will give translators context for how the string is being used.